frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +258,26 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    local info = Notification:new{text = _("Changing font size")}
+    UIManager:show(info)
+    UIManager:forceRePaint()
+    UIManager:nextTick(function()
+        self:onChangeSize("decrease")
+        UIManager:close(info)
+    end)
+    return true
+end
+
+function ReaderFont:onSpread()
+    local info = Notification:new{text = _("Changing font size")}
+    UIManager:show(info)
+    UIManager:forceRePaint()
```

Comment:
Is the `forceRePatin` necessary here? I am under the impression that UIManager will repaint after returned from this method.